README.md

```diff
@@ -1,35 +1,113 @@
 [![PyPI](https://img.shields.io/pypi/v/fairdatapoint-client)](https://pypi.org/project/fairdatapoint-client/)
 [![Build_Test](https://github.com/fair-data/fairdatapoint-client/actions/workflows/build_test.yml/badge.svg)](https://github.com/fair-data/fairdatapoint-client/actions/workflows/build_test.yml)
-[![Codacy Badge](https://app.codacy.com/project/badge/Grade/240d21007f7f41259231141aba79ef1f)](https://www.codacy.com/gh/fair-data/fairdatapoint-client/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=fair-data/fairdatapoint-client&amp;utm_campaign=Badge_Grade)
 [![Coverage Status](https://coveralls.io/repos/github/fair-data/fairdatapoint-client/badge.svg?branch=master)](https://coveralls.io/github/fair-data/fairdatapoint-client?branch=master)
-
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=fair-data_fairdatapoint-client&metric=alert_status)](https://sonarcloud.io/dashboard?id=fair-data_fairdatapoint-client)
 
 # fairdatapoint-client
 
-The `fairdatapoint-client` is a simple and elegant Python package to work with
-REST APIs of FAIR Data Point.
+### Contents
+
+-   [Overview](#overview)
+-   [Installation](#installation)
+-   [Quick Tutorial](#Tutorial)
+-   [Issues & Contributing](#Issues-and-Contributing)
+-   [License](./LICENSE)
+
+## Overview
+
+fairdatapoint-client is a simple and elegant Python package to work with
+REST APIs of [FAIR Data Point](https://github.com/fair-data/fairdatapoint).
+
+The supported APIs are listed below:
+
+| FDP Layers   | Path Endpoint               | Specific Resource Endpoint              |
+|--------------|-----------------------------|-----------------------------------------|
+| fdp          | [baseURL] or [baseURL]/fdp  |                                         |
+| catalog      | [baseURL]/catalog           | [baseURL]/catalog/[catalogID]           |
+| dataset      | [baseURL]/dataset           | [baseURL]/dataset/[datasetID]           |
+| distribution | [baseURL]/distribution      | [baseURL]/distribution/[distributionID] |
 
 ## Installation
 
-To install fairdatapoint-client, do:
+It requires a Python version of 3.7, 3.8 or 3.9.
+
+#### Stable Release
 
-``` {.sourceCode .console}
+The fairdatapoint-client is available on [PyPI](https://pypi.org/project/fairdatapoint-client/):
+
+-   Install the module `pip install fairdatapoint-client`
+
+#### Development Version
```

Comment:
Maybe move this into the Contributing section? I've had users install an unreleased in-development version of MUSCLE3 from GitHub because they wanted to have "the latest version" and then get problems because there are new and unreleased features in there and the examples don't work. So now I'm going to add an explicit note _not_ to install from git there.

Reply:
I'd like to add an explict note to mention that the in-development version might be unstable. User can make his own decision to install from it or not.